website/src/content/docs/internals/changelog.mdx

```diff
@@ -48,17 +48,11 @@ Read our [guidelines for writing a good changelog entry](https://github.com/biom
   describe.skip("test", () => {});
   it.skip("test", () => {});
   ```
-<<<<<<< HEAD
-=======
 
-- Add the rule [noFocusedTests](https://biomejs.dev/linter/rules/no-focused-tests), to disallow skipped tests:
+#### Bug fixes
 
-  ```js
-  describe.only("test", () => {});
-  it.only("test", () => {});
-  ```
```

Comment:
The changelog line should be kept though. We only need to delete the accidentals `<<<<<<< HEAD`, etc.

Reply:
Yep you are correct. But seems like a fix already has been merged to main. Will rebase :+1: